src/rollbar.php

```diff
@@ -480,10 +480,15 @@ protected function build_request_data() {
 
     protected function scrub_request_params($params) {
         $scrubbed = array();
+        $potential_regex_filters = array_filter($this->scrub_fields, function($field) {
+            return strpos($field, '/') === 0;
+        });
         foreach ($params as $k => $v) {
-            if (in_array($k, $this->scrub_fields)) {
-                $count = is_array($v) ? count($v) : strlen($v);
-                $scrubbed[$k] = str_repeat('*', $count);
+            if ($this->_key_should_be_scrubbed($k, $potential_regex_filters)) {
+                $scrubbed[$k] = $this->_scrub($v);
+            } elseif (is_array($v)) {
```

Comment:
If I'm reading this correctly, the behavior here is:

- If the key matches a scrub pattern, then scrub it, no matter what its value is (i.e. even if the value is an array)
- Otherwise, if it's an array, recursively scrub params in that array

I think that sounds like the right thing to do, but I just thought I'd shine some light. Perhaps there should be a test for this?